docs/guide/events/reading/aggregating.md

```diff
@@ -1 +1,133 @@
-**TODO: Add here detailed information about the online aggregation process. I think that it should be first than reading&querying events as normally you're reading events to retrieve the entity state.**
+# Stream aggregation
+
+In Event Sourcing, the state is stored in events. Events are logically grouped into streams. Streams can be thought of as the entities' representation. Traditionally (e.g. in relational or document approach), each entity is stored as a separate record.
```

Comment:
Suggestion: rather than repeating "In Event Sourcing" for every paragraph, you could do something like below:

In event sourcing:
- paragraph 1
- paragraph 2

There also seems to be some repetition of details in the paragraphs. Some consolidation could help to add clarity.